www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,284 @@ function borg_project_metrics_cron() {
+
+  // Get the last total_to_date date recorded.
+  $last_date = $metrics_data['date'];
+
+  // Query github API for $type since $last_date and count them.
+  $results = _get_metrics($type, $last_date);
+
+  $num = 0;
+  foreach ($results as $r) {
+    $paginated_metrics = json_decode($r);
+    foreach ($paginated_metrics as $metric) {
+      if (!empty($metric)) {
+        $num++;
+      }
+    }
+  }
+  $metrics = $num;
+
+  // Get timestamp.
+  date_default_timezone_set('UTC');
+  $now = date('Y-m-d H:i:s');
+  $total_to_date = $metrics_total + $metrics;
+
+  // Record the metrics in the DB.
+  _put_metrics_into_db($type, $now, $total_to_date);
+}
+
+/**
+ * Implements hook_block_info().
+ */
+function borg_project_metrics_block_info() {
+  $blocks['backdrop_project_metrics'] = array(
+    'info' => t('Backdrop Project Metrics'),
+    'description' => t('The number of users, commits, and comments to Backdrop since the project started.'),
+  );
+
+  return $blocks;
+}
+
+/**
+ * Implements hook_block_view().
+ */
+function borg_project_metrics_block_view($delta = '', $settings = array()) {
+  $block = array();
+
+  // Get the metrics.
+  $users = _get_display_metric_from_db('users')['total_to_date'];
+  $commits = _get_display_metric_from_db('commits')['total_to_date'];
+  $comments = _get_display_metric_from_db('comments')['total_to_date'];
+
+  // Path to images directory.
+  $images_path = backdrop_get_path('module', 'borg_project_metrics');
+
+  // Block content.
+  $metrics_content = <<<HTML
+    <div class="project-metrics">
+      <div class="users-metric">
+        <p class="users-img"> <img src="$images_path/images/users.svg" width="66" /> </p>
+        <p class="users-number"> $users </p>
+        <p class="users-label"> users </p>
+      </div>
+      <div class="commits-metric">
+        <p class="commits-img"> <img src="$images_path/images/commit.svg" width="66" /> </p>
+        <p class="commits-number"> $commits </p>
+        <p class="commits-label"> commits </p>
+      </div>
+      <div class="comments-metric">
+        <p class="comments-img"> <img src="$images_path/images/comments.svg" width="66" /> </p>
+        <p class="comments-number"> $comments </p>
+        <p class="comments-label"> comments </p>
+      </div>
+    </div>
+    <div class="join-the-fun">
+      <p>
+        Join the fun!
+        <a href="https://github/backdrop/backdrop-issues/issues">
+          backdrop/backdrop
+        </a>
+      </p>
+    </div>
+HTML;
+  switch ($delta) {
+    case 'backdrop_project_metrics':
+      $block['subject'] = NULL;
+      $block['content'] = $metrics_content;
+      return $block;
+  }
+}
+
+/**
+ * Helper function to retrieve metric form DB for backdrop_project_metrics block.
+ *
+ * @param string $type
+ * The type of metric to retrieve: {commits, comments, users}.
+ *
+ * @return array $metric
+ * An array of the db fields of the most recent $type metric.
+ */
+function _get_display_metric_from_db(string $type) {
```

Comment:
we should properly namespace these helper functions.  `borg_project_metrics_load_from_db` or something.